viral/correct_2p_sessions.py

```
@@ -400,15 +439,195 @@ def jb032_2025_04_10(c: SessionCorrection) -> SessionCorrection:
+    return SessionCorrection(
+        epochs=c.epochs,
+        all_tiff_timestamps=c.all_tiff_timestamps,
+        stack_lengths_tiffs=c.stack_lengths_tiffs,
+        chunk_lengths_daq=c.chunk_lengths_daq,
+        frame_times_daq=c.frame_times_daq,
+        offset_after_pre_epoch=0,
+    )
+
+
+@register_correction("JB032", "2025-04-10")
+def jb032_2025_04_10(c: SessionCorrection) -> SessionCorrection:
```

Ruff (0.14.0)

Redefinition of unused `jb032_2025_04_10` from line 412

(F811)